ckanext/geocat/tests/test_distribution_metadata.py

```diff
@@ -35,81 +36,78 @@ def _load_xml(self, filename):
     def _is_multi_lang(self, value):
         for lang in ['de', 'fr', 'it', 'en']:
             self.assertIn(lang, value)
-        
-    def test_fields(self):
+
+    def test_all_resources(self):
         xml = self._load_xml('complete.xml')
-        dataset = self.csw_map.get_metadata(xml, self.geocat_identifier)
+        dataset = self.csw_map.get_metadata(xml, self.geocat_identifier_deprecated_protocols)
         distributions = dataset.get('resources')
```

Comment:
definition of xml, dataset and distributions is the same for all tests, would it make sense to move it to separate function or make a distribution class object?